src/libraries/System.Private.CoreLib/src/System/Numerics/Vector_1.cs

```diff
@@ -803,5 +803,285 @@ public bool TryCopyTo(Span<T> destination)
+#pragma warning restore CS8500 // This takes the address of, gets the size of, or declares a pointer to a managed type ('T')
+
+        /// <inheritdoc cref="ISimdVector{TSelf, T}.StoreUnsafe(TSelf, ref T)" />
+        static void ISimdVector<Vector<T>, T>.StoreUnsafe(Vector<T> vector, ref T destination) => vector.StoreUnsafe(ref destination);
+
+        /// <inheritdoc cref="ISimdVector{TSelf, T}.StoreUnsafe(TSelf, ref T, nuint)" />
+        static void ISimdVector<Vector<T>, T>.StoreUnsafe(Vector<T> vector, ref T destination, nuint elementOffset) => vector.StoreUnsafe(ref destination, elementOffset);
+
+        /// <inheritdoc cref="ISimdVector{TSelf, T}.Subtract(TSelf, TSelf)" />
+        static Vector<T> ISimdVector<Vector<T>, T>.Subtract(Vector<T> left, Vector<T> right) => left - right;
+
+        /// <inheritdoc cref="ISimdVector{TSelf, T}.Sum(TSelf)" />
+        static T ISimdVector<Vector<T>, T>.Sum(Vector<T> vector) => Vector.Sum(vector);
+
+        /// <inheritdoc cref="ISimdVector{TSelf, T}.ToScalar(TSelf)" />
+        static T ISimdVector<Vector<T>, T>.ToScalar(Vector<T> vector) => vector.ToScalar();
+
+        /// <inheritdoc cref="ISimdVector{TSelf, T}.Truncate(TSelf)" />
+        static Vector<T> ISimdVector<Vector<T>, T>.Truncate(Vector<T> vector) => Vector.Truncate(vector);
+
+        /// <inheritdoc cref="ISimdVector{TSelf, T}.TryCopyTo(TSelf, Span{T})" />
+        static bool ISimdVector<Vector<T>, T>.TryCopyTo(Vector<T> vector, Span<T> destination) => vector.TryCopyTo(destination);
+
+        /// <inheritdoc cref="ISimdVector{TSelf, T}.WithElement(TSelf, int, T)" />
+        static Vector<T> ISimdVector<Vector<T>, T>.WithElement(Vector<T> vector, int index, T value) => vector.WithElement(index, value);
+
+        /// <inheritdoc cref="ISimdVector{TSelf, T}.Xor" />
+        static Vector<T> ISimdVector<Vector<T>, T>.Xor(Vector<T> left, Vector<T> right) => left ^ right;
+
+        //
+        // New Surface Area
+        //
+
+        static bool ISimdVector<Vector<T>, T>.AnyWhereAllBitsSet(Vector<T> vector) => Vector.EqualsAny(vector, AllBitsSet);
+
+        static bool ISimdVector<Vector<T>, T>.Any(Vector<T> vector, T value) => Vector.EqualsAny(vector, Vector.Create(value));
+
+        static int ISimdVector<Vector<T>, T>.IndexOfLastMatch(Vector<T> vector)
+        {
+            if (sizeof(Vector<T>) == 64)
+            {
+                ulong mask = vector.AsVector512().ExtractMostSignificantBits();
+                return 63 - BitOperations.LeadingZeroCount(mask); // 63 = 64 (bits in Int64) - 1 (indexing from zero)
+            }
+            else if (sizeof(Vector<T>) == 32)
+            {
+                uint mask = vector.AsVector256().ExtractMostSignificantBits();
+                return 31 - BitOperations.LeadingZeroCount(mask); // 31 = 32 (bits in Int32) - 1 (indexing from zero)
+            }
+            else
```

Comment:
With the support you added for controlling the size of a `Vector<T>` and with SVE, it's not possible this could be a value besides 16, 32, or 64?

Reply:
`Vector<T>` can only be a power of 2 that is at least 16-bytes. The JIT has no support for anything other than 16/32/64 today and SVE will be shipping as experimental + limited to 128-bits in .NET 9.

I need to properly implement these helpers in the JIT, but that was out of scope for what I wanted to get done in this PR since it's more complex.